recipe/meta.yaml

```diff
@@ -31,6 +31,7 @@ requirements:
     - scipy !=1.4.0
     - sqlalchemy >=1.1.0
     - tqdm
+    - PyYAML
```

Comment:
Added to align with our [setup.py](https://github.com/optuna/optuna/blob/release-v2.9.0/setup.py#L40).